% !TeX document-id = {55cdd1f2-c708-43a6-9dcf-ca7bc9e91f17}
% !BIB program = biber
\documentclass[10pt]{beamer}

\usetheme{scilifelab}
\usepackage{appendixnumberbeamer}

\usepackage{booktabs}
\usepackage[scale=2]{ccicons}

\usepackage{hyperref}
\hypersetup{colorlinks=true, linkcolor=scAqua, urlcolor=scAqua, citecolor=scAqua}

\usepackage[backend=biber,style=apa, sortcites=true,sorting=nyt]{biblatex}
\addbibresource{literature/base-odyssey.bib}


\usepackage{pgfplots}
\usepgfplotslibrary{dateplot}

\usepackage{xspace}
\newcommand{\themename}{\textbf{\textsc{scilifelab}}\xspace}
\newcommand{\credit}[1]{{\vspace{\fill} \par \raggedleft \scriptsize \mdseries \color{mDarkBrown} #1 \par}}
\newcommand{\creditdark}[1]{{\vspace{\fill} \par \raggedleft \scriptsize \mdseries \color{scMGray} #1 \par}}
\newcommand{\creditdarknofill}[1]{{\par \raggedleft \scriptsize \mdseries \color{scMGray} #1 \par}}
\newcommand{\creditleft}[1]{{\vspace{\fill} \par \raggedright \scriptsize \mdseries \color{mDarkBrown} #1 \par}}
\newcommand{\creditdarkleft}[1]{{\vspace{\fill} \par \raggedright \scriptsize \mdseries \color{scMGray} #1 \par}}
\newcommand{\creditdarkleftnofill}[1]{{\par \raggedright \scriptsize \mdseries \color{scMGray} #1 \par}}
\newcommand{\citeme}[1]{{\xspace\color{scAqua} \scriptsize [\cite{#1}]}}
\newcommand{\feature}[1]{{\color{scLime} \textbf{#1}}}
\newcommand{\remark}[1]{{\par \color{scGrape} \ensuremath{\rightarrow} \emph{#1}}}

\makeatletter
\newcommand*{\myroman}[1]{{\fontfamily{ptm}\selectfont \expandafter\@slowromancap\romannumeral #1@}}
\makeatother

\title{2001: A Base Odyssey}
\subtitle{The era of genomics and massive parallel sequencing}
\date{February 24, 2025}
\author{Matthias Zepper, PhD}
\institute{NGI Stockholm\par \href{https://ngisweden.scilifelab.se}{https://ngisweden.scilifelab.se}}
\titlegraphic{\hfill\includegraphics[height=1cm]{./additional_graphics/SciLifeLab_Logotype_Green_POS.png}}

\begin{document}

\maketitle

\begin{frame}{2001: Draft assemblies of the human genome are published}
	\begin{figure}
		\includegraphics[width=0.8\textwidth]{figures/humangenomeproject.png}
		\caption{The private company Celera\citeme{Venter2001} and the International Human Genome Sequencing Consortium\citeme{Lander2001} both publish a draft sequence of the euchromatic portion of the human genome.}
	\end{figure}
	\credit{doi: 10.1126/science.1058040; doi: 10.1038/35057062}
\end{frame}

\begin{frame}[standout]{The overture to the genomic era}
	\begin{figure}
		\hspace*{-1.1cm}
		\includegraphics[width=1.25\textwidth]{./additional_graphics/Opening_2001-_A_Space_Odyssey.png}
		\creditdarknofill{A remake of the opening scene by SumoSebi, \href{https://commons.wikimedia.org/wiki/File:Opening_2001-_A_Space_Odyssey.png}{CC-BY-SA on Wikimedia Commons}}
	\end{figure}
	{\small	Stanley Kubrick's \emph{2001- A Space Odyssey} premieres  2 April 1968}

\end{frame}

\begin{frame}{1968: Nobel prize for the interpretation of the genetic code}
	\begin{columns}[T,onlytextwidth]
		\column{0.6\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/nobel1968.png}
		\end{figure}
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/codesonne2.png}
		\end{figure}
	\end{columns}
	\begin{itemize}
		\item The genetic code is (almost) universal$^{[1]}$ 
		\item It was resolved entirely using synthetic sequences.
	\end{itemize}
	 \par {\scriptsize [1] http://www.ncbi.nlm.nih.gov/Taxonomy/taxonomyhome.html/index.cgi?chapter=tgencodes}
	\credit{https://www.nobelprize.org/prizes/medicine/1968/summary/ | User:Mouagip, Wikimedia Commons, PD}
\end{frame}

\begin{frame}{Encoded information of naturally occuring DNA unknown}
	\begin{columns}[T,onlytextwidth]
		\column{0.3\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/sanger.png}
		\end{figure}
		\column{0.7\textwidth}
		\vspace{3em}
		\begin{itemize}
			\item Peptides could be sequenced since the 
			1950s (Sanger method, Edman degradation).
			\item Sequencing of DNA was one of the most urgent, unresolved problems in the early 1970s.
			\item Frederick Sanger (Nobel laureate for sequencing Insulin 1958) started working with DNA. 
		\end{itemize}
	\end{columns}
\end{frame}

\begin{frame}{1977: Chain-termination sequencing by Frederick Sanger}
	\begin{columns}[T,onlytextwidth]
		\column{0.35\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/sangersequencing.png}
		\end{figure}
		\column{0.65\textwidth}
		\begin{itemize}
			\item DNA fragments could be separated by size.
			\item Sanger's method creates sequence-derived length patterns.
			\item It relies on radioactive labeling and in-vitro amplification of DNA.
		\end{itemize}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/sangerpaper.png}
			\caption{\citeme{Sanger1977}}
		\end{figure}
	\end{columns}
	\credit{$\leftarrow$ Fig.5.40, LÃ¶ffler: Biochemie und Pathobiochemie 7th Ed.}
\end{frame}

\begin{frame}{1980: Nobel prize for DNA sequencing}
	\begin{columns}[T,onlytextwidth]
		\column{0.6\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/nobel-sanger.png}
		\end{figure}
		\column{0.5\textwidth}
		\begin{itemize}
			\item Ample DNA input needed\par {\scriptsize PCR was introduced in 1989}
			\item Four reactions per sequence
			\item Read length $\sim$ 200bp
		\end{itemize}
		\vspace{0.2cm}
		\begin{figure}
			\includegraphics[width=0.8\textwidth]{./figures/sangergel.png}
		\end{figure}
	\end{columns}
 \creditleft{https://www.nobelprize.org/prizes/chemistry/1980/summary/}
\end{frame}

\begin{frame}{Advanced Sanger sequencing for the Human Genome Project}
	\begin{columns}[T,onlytextwidth]
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/chromatogramm-eng.png}
		\end{figure}
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{./figures/baseclearseq.jpg}
		\end{figure}
		\vspace{0.2cm}
		\begin{itemize}
			\item Fluorescent chain terminators.
			\item Capillary electrophoresis for size separation of amplicons.
			\item Parallelized and automated.
			\item Sequencing technology of the Human Genome Project (1990-2004).
		\end{itemize}
		\par \vspace{0.5cm}
		\credit{$\leftarrow$ A chromatogram generated by Sanger sequencing}
	\end{columns}
\end{frame}

% % % % % % % % % % % % % % % % % % % % % %  % % % % % % % % % %

\section{Next-generation sequencing}

% % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % % 

\begin{frame}{New high-throughput methods were developed}
	\begin{center}
		 \begin{figure}
		\includegraphics[width=0.7\textwidth]{./figures/sequencingcosts2018eng.pdf}
		\caption{Sequencing costs per one million bases of raw sequence}
		\end{figure}
	\end{center}
    \begin{description}
	\item [1990-2004:] Human Genome Project sequencing: US \$500 million
	\item [2025:] Sequencing of a human genome: $\sim$ US \$100-1000
\end{description}
	\credit{National Human Genome Research Institute (NHGRI) \linebreak \href{https://www.genome.gov/about-genomics/fact-sheets/Sequencing-Human-Genome-cost}{https://www.genome.gov/about-genomics/fact-sheets/Sequencing-Human-Genome-cost}}
\end{frame}

\begin{frame}{Around 2010: Sanger sequencing was outcompeted by NGS}
	\begin{columns}[T,onlytextwidth]
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=0.8\textwidth]{./figures/abi3730xl.jpg} \vspace{1em} \\
		\end{figure}
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=0.8\textwidth]{./figures/system-carousel-hiseq2500-left.png}
		\end{figure}
	\end{columns}
	\begin{columns}[T,onlytextwidth]
		\column{0.5\textwidth}
		\begin{exampleblock}{}
			\textbf{ABI 3730xl DNA Sequencer}\\
			(Sanger Multiplex, 2013)
			\begin{itemize}
				\item $\sim$6912 reads of 400bp
				\item $\sim$2,76 Mbp / day
			\end{itemize}
		\end{exampleblock}
		\column{0.5\textwidth}
		\begin{exampleblock}{}
			\textbf{Illumina HiSeq 2500}  \vspace{0.3em} \\
			(NGS / MPS, 2013)
			\begin{itemize}
				\item $\sim$600 Million reads of 100bp
				\item $\sim$60.000 Mbp / day
			\end{itemize}
			{ \small (depending on settings and sequencing chemistry used)}
		\end{exampleblock}
	\end{columns}
\end{frame}

% % % % % % % % % % % % % % % % % % % % % % % 

\section{National Genomics Infrastructure Sweden}

% % % % % % % % % % % % % % % % % % % % % % % 

\begin{frame}{DNA sequencing facilities provide sequencing capacity}
	\begin{center}
	\hspace*{-1cm}
	\includegraphics[height=0.2\textheight]{./additional_graphics/NGI-logo.png}
	\end{center}
	\begin{itemize}
		\item DNA sequencing of paramount importance for life science.
		\item 2013: National Genomics Infrastructure Sweden is founded.
		\item Our mission is to offer a state-of-the-art infrastructure available to researchers all over Sweden.
     \end{itemize}
\end{frame}

\begin{frame}{Users of the National Genomics Infrastructure Sweden}
	\begin{center}
		\hspace*{-1cm}
		\includegraphics[height=0.7\textheight]{./figures/ngi-users-2024.png}
	\end{center}
	\credit{\href{https://ngisweden.scilifelab.se/resources/ngi-stockholm-status/}{https://ngisweden.scilifelab.se/resources/ngi-stockholm-status/}}
\end{frame}

\begin{frame}{National Genomics Infrastructure Sweden}
	\begin{center}
		\hspace*{-1cm}
		\includegraphics[height=0.2\textheight]{./additional_graphics/NGI-logo.png}
	\end{center}
	\begin{itemize}
		\item NGI is a sequencing facility for \emph{research projects} 
		\item Part of the Genomics Platform at SciLifeLab
		\item Distributed in 3 nodes:
		\begin{itemize}
			\item SNP\&SEQ Technology Platform, Uppsala
			\item Uppsala Genome Center
			\item NGI Stockholm + Eukaryotic Single Cell Genomics (ESCG), Solna 
		\end{itemize}
	\end{itemize}
	\credit{\href{https://ngisweden.scilifelab.se}{https://ngisweden.scilifelab.se}}
\end{frame}

\begin{frame}{NGI-S employs various sequencing technologies}
	\begin{center}
		\includegraphics[width=0.7\textwidth]{./figures/ngi-choi-flowcell.jpg} \\
		\hspace*{-1cm}
		\includegraphics[width=1.2\textwidth]{./figures/ngis-throughput-2024.png}
	\end{center}
	\begin{itemize}
		\item In 2024, NGI Stockholm sequenced on average 1200 Gbp/day
	\end{itemize}
	\credit{\href{https://ngisweden.scilifelab.se/resources/ngi-stockholm-status/}{https://ngisweden.scilifelab.se/resources/ngi-stockholm-status/}}
\end{frame}

% % % % % % % % % % % % % % % % % % % % % % % 

\section{Sequencing platforms}

% % % % % % % % % % % % % % % % % % % % % % % 

\begin{frame}{Sequencing platforms / technologies since Sanger}
	%\metroset{block=fill}
	\begin{exampleblock}{Next generation sequencing}
		\begin{itemize}
			\item Roche 454 sequencing (Pyrosequencing)
			\item Ion semiconductor sequencing
			\item \textbf{Illumina (Solexa) sequencing}
			\item \textbf{PacBio HiFi Sequencing}
		\end{itemize}
	\end{exampleblock}
	\begin{alertblock}{Third generation sequencing}
		\begin{itemize}
			\item \textbf{Oxford Nanopore sequencing}
			\item \textbf{Element Biosciences Avidite Sequencing}
			\item Ultima Genomics UG 100 Sequencing
			\item MGI DNBSEQ Technology
			\item Singular Genomics G4X
		\end{itemize}
	\end{alertblock}
\credit{Platforms in \textbf{bold} are in use at the National Genomics Infrastructure}
\end{frame}

\begin{frame}{Sequencing platforms / technologies since Sanger}
	%\metroset{block=fill}
	\begin{exampleblock}{Sequencing by synthesis}
		\begin{itemize}
			\item Roche 454 sequencing (Pyrosequencing)
			\item Ion semiconductor sequencing
			\item \textbf{Illumina (Solexa) sequencing}
			\item \textbf{PacBio HiFi Sequencing}
			\item \textbf{Element Biosciences Avidite Sequencing}
			\item Ultima Genomics UG 100 Sequencing
			\item MGI DNBSEQ Technology
			\item Singular Genomics G4X
		\end{itemize}
	\end{exampleblock}
	\begin{alertblock}{Direct DNA/RNA sequencing}
		\begin{itemize}
			\item \textbf{Oxford Nanopore sequencing}
		\end{itemize}
	\end{alertblock}
\credit{Platforms in \textbf{bold} are in use at the National Genomics Infrastructure}
\end{frame}

\begin{frame}{Illumina sequencing is \emph{the} NGS sequencing platform}
	\begin{center}
		\includegraphics[width=0.7\textwidth]{./figures/ngi-choi-novaseqxplus.jpg} \\
		\hspace*{-1cm}
		\includegraphics[width=1.2\textwidth]{./figures/ngis-throughput-2024-illumina.png}
	\end{center}
\creditleft{Illumina's sequencing by synthesis technology is NGI's bread-and-butter platform}
\end{frame}

\begin{frame}[standout]{Search engines start failing to find meaningful content}
	\begin{columns}[T,onlytextwidth]
		\hspace*{-0.7cm} 
		\column{0.5\textwidth}
		\begin{figure}
			\includegraphics[width=\textwidth]{figures/library-prep.png}
			\caption{Sequencing lib}
		\end{figure}
		\column{0.5\textwidth}
			\begin{alertblock}{Direct DNA/RNA sequencing}
			\begin{itemize}
				\item \textbf{Oxford Nanopore sequencing}
			\end{itemize}
		\end{alertblock}
	\end{columns}
\end{frame}

\begin{frame}[standout]{Milestones in DNA sequencing history}
	\vspace*{-1.1cm}
	\begin{center}
		\hspace*{-1.1cm}
		\includegraphics[width=1.2\textwidth]{./figures/timeline.png}
	\end{center}
	\creditdark{Figure by Anja Mezger}
\end{frame}

\section{References}
\begin{frame}[allowframebreaks]{References}
\begingroup
\renewcommand*{\bibfont}{\footnotesize} 
\printbibliography[heading=none]
\endgroup
\end{frame}

\end{document}
